### ISSUE TITLE
Android - Game Map doesn't refresh on turn

### ISSUE THREAD

**Issue by jwwoody** [byucs340ta/Spring2019#44]
The game map doesn't refresh on your turn. For example, on the turn that I acquired enough purple cards the route for San Francisco to Los Angeles was unavailable. Only when I clicked on Game info and back to Game Map did the route appear.
<img width="770" alt="lasf1" src="https://user-images.githubusercontent.com/51487814/59139929-65d18980-8954-11e9-9a70-1c46c785dc8b.PNG">
<img width="817" alt="lasf3" src="https://user-images.githubusercontent.com/51487814/59139938-6ec25b00-8954-11e9-9a57-8462c3677599.PNG">

<img width="190" alt="lasfBeforeRefresh" src="https://user-images.githubusercontent.com/51487814/59139946-7d107700-8954-11e9-848f-a5e43def8347.PNG">
<img width="205" alt="lasfafter" src="https://user-images.githubusercontent.com/51487814/59139947-80a3fe00-8954-11e9-8e61-b43066404f77.PNG">

**Comment by Bobleyl** [comment]
Yeah, we knew about this one coming in.  The spinner only updates when you either refresh the fragment or go to a different tab and then come back.  We will need to fix that by Monday.
